api.js

```
@@ -0,0 +1,5 @@
+import axios from 'axios';
+
+export const login = (email, password) => {
+  return axios.post('/api/login', { email, password });
```

Consider handling errors at this level as well. You could throw an error here and then catch it in the LoginPage component to set the error state. This would make the API function more robust.